contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM resin/rpi-raspbian:jessie
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libsqlite3-dev libsystemd-journal-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.4.3
```

Comment:
This now needs updating in line with arm/jessie https://github.com/docker/docker/commit/f32ccb080ac382a386cbe33ed1d11048474206c5#diff-0eafa589a244859aba06e5c8a1b522fc